salt/modules/apache.py

```diff
@@ -404,14 +404,13 @@ def _parse_config(conf, slot=None):
         else:
             print('{0}'.format(conf), file=ret, end='')
     elif isinstance(conf, list):
-        for value in conf:
-            print(_parse_config(value, str(slot)), file=ret)
+        print('{0} {1}'.format(slot, ' '.join(conf)), file=ret, end='')
```

Comment:
Why do we have a print in here at all? Execution modules should not be printing to the console at all. If anything, let's change this to a `log.debug` or `log.info` message.